conf/schemas/cloudtrail.json

```diff
@@ -0,0 +1,67 @@
+{
+	"cloudtrail:digest": {
+    "schema": {
+      "awsAccountId": "string",
+      "digestEndTime": "string",
+      "digestPublicKeyFingerprint": "string",
+      "digestS3Bucket": "string",
+      "digestS3Object": "string",
+      "digestSignatureAlgorithm": "string",
+      "digestStartTime": "string",
+      "logFiles": [],
+      "newestEventTime": "string",
+      "oldestEventTime": "string",
+      "previousDigestHashAlgorithm": "string",
+      "previousDigestHashValue": "string",
+      "previousDigestS3Bucket": "string",
+      "previousDigestS3Object": "string",
+      "previousDigestSignature": "string"
+    },
+    "parser": "json"
+  },
+	"cloudtrail:events": {
```

Comment:
Spacing is weird here

Reply:
VSCode exploded trying to format this file. I fixed it though.